app/controllers/api/v1/medical_shifts_controller.rb

```diff
@@ -45,6 +45,18 @@ def update
         end
       end
 
+      def destroy
+        authorize(medical_shift)
+
+        result = MedicalShifts::Destroy.result(id: medical_shift.id.to_s)
+
+        if result.success?
+          render json: { message: "MedicalShift deleted successfully." }, status: :ok
```

Comment:
good job! here we can apply i18n to this message, but this can be left for another activity without any problems.

Reply:
It was added in the other [PR](https://github.com/espoo-dev/med_system_backend/pull/208) in this commit 33e5ad533efd4b9a1e9d004e31cd677eff0574cf. After all merged, I'll fix this render mesage.